packages/workflows/src/api/ArchivedDelete.ts

```diff
@@ -0,0 +1,40 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+
+import { AbstractSession } from "@brightside/imperative";
+import { ZosmfRestClient } from "../../../rest";
+import { WorkflowConstants, nozOSMFVersion, noWorkflowKey } from "./WorkflowConstants";
+import { WorkflowValidator } from "./WorkflowValidator";
+
+/**
+ * Class to handle deletion of workflow from an archived z/OSMF workflow from a z/OS system.
+ */
+export class ArchivedDeleteWorkflow {
+
+    /**
+     * Delete a workflow from an archived
+     * @param {AbstractSession} session                     - z/OSMF connection info
+     * @param {string} workflowKey                          - Unique identifier of the workflow instance.
+     * @param {string} zOSMFVersion                         - Identifies the version of the zOSMF workflow service.
+     * @returns {string}
+     */
+ public static async archivedDeleteWorkflow(session: AbstractSession, workflowKey: string,
```

Comment:
can we have 4 spaces of indentation ? : )

Reply:
done